### PR TITLE
Replace JarSigner with ApkSigner

### PR DESCRIPTION
**Pull request by pavi2410**
Docs:
  - https://developer.android.com/studio/command-line/apksigner
  - https://developer.android.com/studio/publish/app-signing#signing-manually

### CODE REVIEW

File: appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java [hunk 1]
```diff
@@ -950,18 +950,18 @@ public static boolean compile(Project project, Set<String> compTypes,
...
-
     // ZipAlign the apk file
     out.println("________ZipAligning the apk file");
     if (!compiler.runZipAlign(apkAbsolutePath, tmpDir)) {
       return false;
     }
 
+    // Sign the apk file
+    out.println("________Signing the apk file");
+    if (!compiler.runJarSigner(apkAbsolutePath, keystoreFilePath)) {
```
Comment (pavi2410) [inline appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java]:
```suggestion
    if (!compiler.runApkSigner(apkAbsolutePath, keystoreFilePath)) {
```

**Comment by ewpatton** [comment]
I believe that @jisqyv has already done this on another branch that hasn't been published yet.

**Comment by jisqyv** [comment]
@pavi2410 @ewpatton I've been doing the apksigning by hand. I'll test this change and likely merge it.

**Comment by ewpatton** [comment]
@jisqyv Sounds good to me.

**Comment by jisqyv** [comment]
@pavi2410 I'm getting ready to merge this info our sources. But first I'm going to push another commit onto your feature branch, merging in master, fixing the conflicts. etc. This way I can merge the code from GitHub keeping you as the author of the change.

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Pull request merged**